ting_material_details.field.inc

```diff
@@ -457,6 +463,36 @@ function ting_material_details_field_formatter_view($entity_type, $entity, $fiel
             '#markup' => $entity->serieDescription,
           );
           break;
+
+        case 'ting_details_is_part_of_search':
+          $type = $uri = NULL;
+          foreach ($entity->getRelations() as $relation) {
+            $type = explode(':', $relation->getType());
+            if ( isset($type[1]) && $type[1] == 'isPartOfManifestation' ) {
+              $uri  = $relation->getURI();
+            }
```

Comment:
Why is the `explode()` necessary here? In the ting_relation module [we do a similar check for "isPartOfManifestation"](https://github.com/ding2/ting_relation/blob/master/includes/ting_relation.field.inc#L78) and here there is no explode.

Also there are a few whitespace issues here.